src/panel/favorites_panel.js

```diff
@@ -27,13 +33,41 @@ function Favorite() {
   PanelManager.register(this)
 }
 
+Favorite.prototype.toggleMore = function (position) {
+  if(this.openMoreMenuPosition === -1) {
+    this.openMoreMenu(position)
+  } else {
+    this.closeMoreMenu()
+  }
+}
+
+Favorite.prototype.openMoreMenu = function (position) {
+  this.openMoreMenuPosition = position
+  let menu = document.querySelector(`#favorite_more_${position}`)
+  menu.classList.add('favorite_panel__item__more--active')
+}
+
+Favorite.prototype.closeMoreMenu = function () {
+  let menu = document.querySelector(`#favorite_more_${this.openMoreMenuPosition}`)
+  if(menu) {
+    menu.classList.remove('favorite_panel__item__more--active')
+    this.openMoreMenuPosition = -1
+  }
+}
+
+Favorite.prototype.openShare = function (poi) {
+  let url = `${window.location}${poi.id}`
+  this.sharePanel.open(url)
+}
+
 Favorite.prototype.isDisplayed = function () {
   return this.displayed
 }
 
 Favorite.prototype.toggle = function() {
   if(this.active) {
     this.close()
+    this.openMoreMenuPosition = -1
```

Comment:
I would put that in the `open` function. The favorite panel may be closed via other contexts.